src/visualizations/features/VisualizationPicture.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { render, screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import VisualizationPicture from "../features/VisualizationPicture";
+import { VisualizationPicture_VisualizationFragment } from "./VisualizationPicture.generated";
+
+describe("VisualizationPicture", () => {
+  const visualization: VisualizationPicture_VisualizationFragment = {
```

Comment:
You don't need to type it as `VisualizationPicture_VisualizationFragment`
```suggestion
  const visualization = {
```

Reply:
Okay